_pages/Mentoronto.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: Mentoronto
+pageUrl: https://mentoronto.github.io/
```

Comment:
CI tests are failing because of "https://"
this line should be just mentoronto.github.io
I had the same error.
Cheers :)